### ISSUE TITLE
ark backup create issue.

### ISSUE THREAD

**Issue by ShadoWalker898** [heptio/ark#142]
Hi Folks,

I have installed heptio-ark in gcp kubernetes cluster. But when I execute "ark backup create nginx-backup --selector app=nginx" i get below errors. Can someone please help.

PS: Im in Ark root dir.

[root@shadownet ark-master]# ark backup create nginx-backup --selector app=nginx
dirname: missing operand
Try 'dirname --help' for more information.
basename: missing operand
Try 'basename --help' for more information.
WARNING: IPv4 forwarding is disabled. Networking will not work.
An error occurred: invalid configuration: no configuration has been provided

**Comment by ncdc** [comment]
@ShadoWalker898 it sounds like you don't have your `KUBECONFIG` environment variable configured. Could you please try setting that appropriately and see if the error goes away?

**Comment by ShadoWalker898** [comment]
I executed below command but nothing returned.

alias ark='docker run --rm -u $(id -u) -v $(dirname $KUBECONFIG):/kubeconfig -e KUBECONFIG=/kubeconfig/$(basename $KUBECONFIG) gcr.io/heptio-images/ark:latest'

and KUBECONFIG environment variable configured as below

[root@shadownet ~]# echo $HOME/.kube/config
/root/.kube/config

**Comment by ncdc** [comment]
What does ‘echo $KUBECONFIG’ show?

**Comment by ShadoWalker898** [comment]
It shows nothing. It's empty...

**Comment by ncdc** [comment]
You need to set it.

**Comment by ShadoWalker898** [comment]
Its like this. I have setup google cloud sdk shell in a centos vm. And from it i setup heptio-ark. So how to set the $KUBECONFIG path ?

**Comment by ncdc** [comment]
`export KUBECONFIG=$HOME/.kube/config`

Alternatively, you can download the source code and compile ark yourself:
```
git clone https://github.com/heptio/ark
git checkout v0.4.0
make
```
At this point, you'll have `_output/bin/ark` that you can copy wherever you want and run it without using the `alias`. In either v0.5.0 or v0.6.0 we will be producing binaries you can download from the GitHub release, so this will be easier soon.

**Comment by ncdc** [comment]
@ShadoWalker898 we released v0.5.0 yesterday, and you can download binaries from the release page. Please try one out and let us know if it works for you. Thanks!

**Comment by ncdc** [comment]
As we discussed on Slack, you have a `/usr/bin/ark` that is being found in your `$PATH`, but this file comes from some other package and is not Heptio Ark. You'll need to make sure you are invoking the correct executable. If you still need assistance, please feel free to find us on Slack again. Thanks for trying out Ark!
